es6-const-let/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    var greetings = [
+      'hello!',
+      'konichiwa!',
+      'nihao',
+      'bonjour',
+      'gutten tag',
+      'ola'
+    ];
```

Comment:
Please make sure you've converted _all_ variables to use _const_ or _let_, as appropriate.

---

![image](https://user-images.githubusercontent.com/16143771/127085188-116267ef-d766-4c91-885c-87dfad9ba885.png)